table_transforms/rasgo_dedupe/rasgo_dedupe.sql

```diff
@@ -0,0 +1,7 @@
+SELECT
+*
+FROM {{ source_table }}
+QUALIFY ROW_NUMBER() OVER (
+    PARTITION BY {%- for col in natural_key %} {{col}}{{"," if not loop.last else ""}} {%- endfor %}
+    ORDER BY {%- for col in order_col %} {{col}}{{"," if not loop.last else ""}} {%- endfor %} {{order_method}}
+) = 1
```

Comment:
we should specifically say in the docs that once the order by is applied, only the top row will be kept and the rest will be dropped